node/trusted_validators_membership_change.re

```diff
@@ -0,0 +1,45 @@
+open Helpers;
```

Comment:
Overall I would ask that you keep this module inside of `Files` it's not a big deal also files interfaces should not be used for internal interfaces.

That will lead to some duplication, but the interface generated by yojson is really not ideal.

Reply:
Keep one copy in Files and another here